### DISCUSSION TITLE
What does linux specific patches imply in yuzu-pineapple package?

### DISCUSSION THREAD

**Discussion by beyond9thousand** [ScoopInstaller/Scoop#5502] [Help and Questions] [answered]
Am i not meant to install this on my windows machine? A little lost here, thanks in advance!
![image](https://user-images.githubusercontent.com/77363063/236674200-1882a207-e14d-4abe-87b5-50f7b390f303.png)

**Comment by rashil2000** [comment, marked as answer]
You can install it, of course. It's just a fork that contains some patches for Linux - it will likely not make any difference on Windows machine.
